Makefile

```diff
@@ -681,11 +681,15 @@ APIDIFF_OLD_COMMIT ?= $(shell git rev-parse origin/main)
 apidiff: $(GO_APIDIFF) ## Check for API differences
 	$(GO_APIDIFF) $(APIDIFF_OLD_COMMIT) --print-compatible
 
-ALL_VERIFY_CHECKS = licenses boilerplate shellcheck tiltfile modules gen conversions doctoc capi-book-summary diagrams import-restrictions
+ALL_VERIFY_CHECKS = licenses boilerplate shellcheck tiltfile modules gen conversions doctoc capi-book-summary diagrams import-restrictions go-directive
 
 .PHONY: verify
 verify: $(addprefix verify-,$(ALL_VERIFY_CHECKS)) lint-dockerfiles ## Run all verify-* targets
 
+.PHONY: verify-go-directive
+verify-go-directive:
+	TRACE=$(TRACE) ./hack/verify-go-directive.sh -g 1.21
```

Comment:
Let's add a GO_DIRECTIVE_VERSION near to GO_VERSION

Reply:
Instead of having the hardcoded const 1.21 in L691 (deep down the Makefile), let's introduce a new variable named `GO_DIRECTIVE_VERSION=1.21` at the beginning of the Makefile, just below the definition of GO_VERSION and GO_CONTAINER_IMAGE.
If we do in this way, all the go version will be defined in a single place easier to find and maintain